src/pages/CartPage.ts

```diff
@@ -1,59 +1,57 @@
 import { expect, test, type Locator, type Page } from '@playwright/test';
-import { AbstractPage } from './BasePage';
+import { BasePage } from './BasePage';
+import { ROUTES } from './_Constants';
  
-export class CartPage extends AbstractPage {
-  readonly page: Page;
+export class CartPage extends BasePage {
+  url: string = ROUTES.cart;
```

Comment:
you can omit type declaration here, it was declared in the base class.

Reply:
fixed